go/mysql/replication_constants.go

```diff
@@ -189,11 +189,6 @@ const (
 	eAnonymousGTIDEvent     = 34
 	ePreviousGTIDsEvent     = 35
 
-	// MySQL 5.7 events
```

Comment:
Even though we don't use these right now, we want to keep them for completeness.
Instead of removing them, we can add linter directives so that staticcheck ignores these.

Reply:
Thank you for your suggestion!
I changed it.